### PR TITLE
Add check for empty comments and notes

### PR DESCRIPTION
**Pull request by Ashesh3**
Builds on https://github.com/coronasafe/care_fe/pull/2525

Adds a check for empty comments and notes from the backend and rejects with proper validation message when needed.

Also some minor formatting change performed automatically by `pre-commit-hooks`

### CODE REVIEW

**Review by sainAk — CHANGES_REQUESTED** [review]
We can do this without regex

File: care/facility/api/serializers/patient.py [hunk 1]
```diff
@@ -347,6 +352,11 @@ class PatientNotesSerializer(serializers.ModelSerializer):
     facility = FacilityBasicInfoSerializer(read_only=True)
     created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
 
+    def validate_empty_values(self, data):
+        if not data.get("note") or re.match(r"^\s*$", data["note"]):
```
[the remark this answers is not in the document]
Comment (Ashesh3) [reply care/facility/api/serializers/patient.py]: Changed to use strip in all 3 places, thanks!

**Review by sainAk — APPROVED** [review]
LGTM

**Comment by vigneshhari** [comment]
I would create a new serializer field that has the empty check validator built-in and use that everywhere instead of re-writing the logic multiple times. Create a PR with this or create an issue to create such a field.

**Review by vigneshhari — APPROVED** [review]
Changes look good, Suggestions for the next iteration provided above, Untested.

**Pull request merged**